runtime/bindings.m

```diff
@@ -88,3 +89,54 @@
 #endif
 #endif
 
+/*
+ * Vector c bindings
+ */
+
+typedef CGPoint (*vision_func) (vector_float2 faceLandmarkPoint, CGRect faceBoundingBox, size_t imageWidth, size_t imageHeight);
+
+CGPoint
+xamarin_CGPoint__VNNormalizedFaceBoundingBoxPointForLandmarkPoint_Vector2_CGRect_nuint_nuint (struct Vector2f faceLandmarkPoint, CGRect faceBoundingBox, xm_nuint_t imageWidth, xm_nuint_t imageHeight) {
+
+	static vision_func func = NULL;
+
+	if (func == NULL) {
+		void *vision_handle = dlopen ("/System/Library/Frameworks/Vision.framework/Vision", RTLD_LAZY);
+		if (vision_handle == NULL)
+			xamarin_assertion_message ("Could not open Vision Framework");
```

Comment:
Good idea, we can add an additional argument that specifies whether the call succeeded or not:

```c
CGPoint
xamarin_CGPoint__VNNormalizedFaceBoundingBoxPointForLandmarkPoint_Vector2_CGRect_nuint_nuint (struct Vector2f faceLandmarkPoint, CGRect faceBoundingBox, xm_nuint_t imageWidth, xm_nuint_t imageHeight, const char **error_msg) {
    error_msg = NULL;
    ...
    if (vision_handle == NULL) {
         *error_msg = "Could not open Vision Framework";
         return CGPointMake (0, 0);
    }
}
```

and then in the caller check for that value:

```csharp
CGPoint pnt;
IntPtr error_msg = NULL;
pnt = xamarin_CGPoint__VNNormalizedFaceBoundingBoxPointForLandmarkPoint_Vector2_CGRect_nuint_nuint (..., ref error_msg);
if (error_msg != IntPtr.Zero)
    throw new SomeException (Marshal.PtrToStringAuto (error_msg));
```